### PR TITLE
Added docs in README 

### PR DESCRIPTION
**Pull request by GalenWong**
includes project start up guide and maintenance related guides.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,146 @@
...
-
-<!-- AUTO-GENERATED-CONTENT:END -->
+<img align=left width=100 src=src/images/acm-hack-logo.svg alt="Hack logo">
+
+# ACM Hack Website
+
+<!-- We do not have netlify yet. We should migrate to it later. -->
+
+
+<!-- [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code-of-conduct.md)
```
Comment (TimothyGu) [inline README.md:8]: Are you sure you don't want to adopt the CoC?

File: README.md [hunk 2]
```diff
@@ -1,97 +1,146 @@
...
+[![Netlify Status](https://api.netlify.com/api/v1/badges/4825a1ca-c270-4bfb-980e-8c616ef8e123/deploy-status)](https://app.netlify.com/sites/acm-hoth/deploys) -->
+\
+\
+This repository contains the code for ACM Hack’s website! 
+
+This website is developed with [Gatsby](https://www.gatsbyjs.org/) and
+[Material UI](https://material-ui.com/), and is a work-in-progress being
```
Comment (TimothyGu) [inline README.md:15]: It's not a work in progress anymore. More like "an active project"

File: README.md [hunk 3]
```diff
@@ -1,97 +1,146 @@
...
+### Troubleshooting
+If something breaks in a weird way, try the following in order:
+- run `yarn`
+- delete the `public`, `.cache`, and `node_modules/.cache` directories
+- delete the `node_modules` directory entirely and reinstall dependencies
+  by running `yarn`
+
+## Maintenance 
+
+### Adding Blogs
```
Comment (TimothyGu) [inline README.md:68]: It's adding **blog posts**. A blog is a website.

File: README.md [hunk 4]
```diff
@@ -1,97 +1,146 @@
...
+
+The blogs that are written in README format.
```
Comment (TimothyGu) [inline README.md:70]: Uh you mean Markdown?

File: README.md [hunk 5]
```diff
@@ -1,97 +1,146 @@
...
+
+
+### Changing Officer Profiles
+
+Officers come and go. We don't stay in college forever. To change the officer
+profiles, go to `src/components/PhotoPage/PhotoPage.js`. Modify the array
+directly.
+
+
+The profile pictures of the officer is put under `src/data/team`. The naming
```
Comment (TimothyGu) [inline README.md:105]: src/images

**Comment by GalenWong** [comment]
@TimothyGu Thanks grammarly

**Comment by TimothyGu** [comment]
They are not grammar things though

**Pull request merged**